mungegithub/mungers/e2e/e2e.go

```diff
@@ -50,7 +51,8 @@ type BuildInfo struct {
 // RealE2ETester is the object which will get status from a google bucket
 // information about recent jobs
 type RealE2ETester struct {
-	NonBlockingJobNames []string
+	Opts                *options.Options
+	NonBlockingJobNames *[]string
```

Comment:
If this value was not a pointer then it would have to be updated with the value from sq.NonBlockingJobNames whenever the option changes. The options module assigns new slices, but the option is bound to sq.NonBlockingJobNames, not RealE2ETester.NonBlockingJobNames since at the time of option registration it is unknown if a fakee2e will be used in place of the reale2etester.